includes/class-edd-db-customers.php

```diff
@@ -424,18 +424,20 @@ public function get_customers( $args = array() ) {
 		if( ! empty( $args['email'] ) ) {
 
 			if( is_array( $args['email'] ) ) {
-				$emails = "'" . implode( "', '", $args['email'] ) . "'";
+
+				$emails_count = count( $args['email'] );
+				$emails_placeholder = array_fill( 0, $emails_count, '%s' );
```

Comment:
We cannot use `0` as the value for the `array_fill`, this was only allowed in PHP 5.6 and greater. Previous versions cannot use `0` as a value for array_fill:

http://php.net/manual/en/function.array-fill.php#refsect1-function.array-fill-changelog
5.6.0	num may now be zero. Previously, num was required to be greater than zero.